### DISCUSSION TITLE
Need more information !

### DISCUSSION THREAD

**Discussion by mahit-dinesh** [community/community#144862] [GitHub Education]
### Hi everyone,

Hi, 

I am trying to submit the documentation via "Take a Picture" option and it wont let me go past this stage in the application process. I dont know how to proceed, there should be an option to upload a document instead of taking a picture which i feel is not easy and lacks clarity of the data need. Please assist. Thanks !

**Comment by IRNCyber** [comment]
Here are some steps you can try to resolve the issue:
 * Check the Application's Specific Guidelines:
   * Refer to the application's instructions: Look for specific guidelines or FAQs about document submission.
   * Contact the application support: Reach out to the application's support team for direct assistance and clarification.
 * Try Different Devices and Browsers:
   * Switch devices: Try using a different device, such as a smartphone or tablet.
   * Change browsers: Experiment with different web browsers like Chrome, Firefox, or Edge.
 * Clear Browser Cache and Cookies:
   * Clear browsing data: This can sometimes resolve technical glitches.
 * Ensure Sufficient Storage and Network Connectivity:
   * Check storage: Make sure your device has enough storage space for the image.
   * Verify network: Ensure a stable internet connection.
 * Check Image Quality and Format:
   * Image quality: The image should be clear and well-lit.
   * Image format: Common formats like JPG or PNG usually work well.
 * Consider Alternative Document Submission Methods:
   * Email: If the application allows, you can email the document directly to the support team.
   * Physical mail: In some cases, physical copies can be mailed.
If the "Take a Picture" option is mandatory and you're facing difficulties, here are some tips for capturing clear images:
 * Good lighting: Ensure the document is well-lit.
 * Steady hand: Use a tripod or place the device on a stable surface.
 * Clear focus: Make sure the document is in sharp focus.
 * Correct angle: Avoid taking pictures at an angle.
 * Crop unnecessary parts: Trim the image to focus on the relevant information.
Remember to be patient and persistent. If you continue to encounter issues, reach out to the application's support team for further assistance.
Would you like to provide more details about the specific application you're using? This might help me provide more tailored advice.
